### ISSUE TITLE
Question about the hyperparameters of SuperGlue

### ISSUE THREAD

**Issue by oyxuan-11** [microsoft/DeBERTa#54]
Hi

What are the parameters of Deberta finetune superglue for each task, such as batch, GPU cards, learning rate, etc.?  
I couldn't find the detailed parameters of each task in SuperGLue in the paper.

How can I reproduce Deberta's results on SuperGlue leaderboard?
